pandevice/base.py

```diff
@@ -1537,6 +1537,31 @@ def delete_type(self, cType):
         for x in objs:
             self.remove(x)
 
+    def request_password_hash(self, value):
```

Comment:
I considered this, but it would mean user scripts would have to do `nearest_pandevice()` before using the function.  I don't think there is any current workflow that a user script would need to execute this function currently..?